src/plugins/expression_metric/README.md

```diff
@@ -0,0 +1,9 @@
+# expressionMetric
+
+Expression Metric plugin adds an `metric` renderer to the expression plugin. @todo write a description.
```

Comment:
@crob611, could you, please, help me with a description of this plugin? Thank you)

Reply:
```suggestion
Expression Metric plugin adds a `metric` renderer and function to the expression plugin. 
```

Reply:
A simple description is fine, I suggested a minor change to yours and that should be fine